modules/n1ql/pages/n1ql-language-reference/advise.adoc

```diff
@@ -395,7 +399,7 @@ Assume that the cost-based optimizer is enabled, and that statistics exist for t
 
 [source,n1ql]
 ----
-ADVISE SELECT * FROM `travel-sample` a WHERE a.type = 'hotel' AND a.country = 'France';
+ADVISE SELECT * FROM `travel-sample`.inventory.hotel a WHERE a.country = 'France';
```

Comment:
Ooo, this is really nice example to get head around advantage/difference of scope/collection usage.